script.js

```diff
@@ -0,0 +1,58 @@
+function selectop() {
+    var operval = document.getElementById("opervalue").value
+    switch (operval) {
+        case '+':
+            sum();
+            break;
+        case "-":
+            subtract();
+            break;
+        case "*":
+            multiply();
+            break;
+        case "/":
+            divide();
+            break;
+        default:
+            badop()
+            break;
```

Comment:
You've got a lot of mixing of lines with and without semi-colons, consistency is the key to clean code.